.github/workflows/docker-build-deploy.yml

```
@@ -31,20 +31,41 @@ jobs:
+              cd ~/telegram_bot
+              docker-compose -f docker-compose.yml config > docker-compose.$(date +%Y%m%d_%H%M%S).yml
+              gzip -d telegram_bot_image.tar.gz
+              docker load < telegram_bot_image.tar
+              docker-compose down
+              if [ $? -ne 0 ]; then
+                  echo "Failed to stop containers" >&2
+                  exit 1
+              fi
+              docker-compose up -d
+              sleep 10
+              docker-compose ps --quiet | xargs docker inspect --format '{{.State.Health.Status}}' | grep -v healthy && exit 1
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)